packages/editor/src/components/post-template/hooks.js

```diff
@@ -41,21 +41,21 @@ export function useAllowSwitchingTemplates() {
 	);
 }
 
-function useTemplates() {
+function useTemplates( postType ) {
 	return useSelect(
 		( select ) =>
 			select( coreStore ).getEntityRecords( 'postType', 'wp_template', {
 				per_page: -1,
-				post_type: 'page',
+				post_type: postType,
 			} ),
 		[]
```

Comment:
Does it make sense to add postType to avoid an eslint warning error regarding a dependency?